python/sports-league-scheduling/tests/test_constraints.py

```diff
@@ -0,0 +1,165 @@
+from timefold.solver.test import ConstraintVerifier
+
+from sports_league_scheduling.domain import *
+from sports_league_scheduling.constraints import *
+
+constraint_verifier = ConstraintVerifier.build(define_constraints, LeagueSchedule, Match)
+
+
+def test_matches_same_day():
+    # Two matches for the home team
+    home_team = Team(id="1", name="TeamA", distance_to_team={})
+    rival_team = Team(id="2", name="TeamB", distance_to_team={})
+    match1 = Match(id="1", home_team=home_team, away_team=rival_team, round=Round(index=0, weekend_or_holiday=False),
+                   classic_match=False)
+    match2 = Match(id="2", home_team=home_team, away_team=rival_team, round=Round(index=0, weekend_or_holiday=False),
+                   classic_match=False)
+    match3 = Match(id="3", home_team=home_team, away_team=rival_team, round=None, classic_match=False)
```

Comment:
`round=None` and `classic_match=False` are the defaults for `Match` and do not need to be specified
(applies to the whole file wherever  `classic_match=False`  or `round=None`).

```suggestion
    match3 = Match(id="3", home_team=home_team, away_team=rival_team)
```